src/Spout/Writer/WriterFactory.php

```diff
@@ -45,4 +45,34 @@ public static function create($writerType)
 
         return $writer;
     }
+
+    /**
+     * @return CSV\Writer
+     */
+    public static function createCSVWriter()
```

Comment:
use these functions in the `create` method above.

Reply:
> method

Should I modify the `create` method?

If needed it could be done later.

Reply:
Yes, please modify the `create` method to call the newly created methods directly.
It should be done now rather than later.